packages/pnpm/package.json

```diff
@@ -150,7 +150,7 @@
     "bundle:pnpm": "cross-var esbuild lib/pnpm.js --bundle --platform=node --outfile=dist/pnpm.cjs --external:node-gyp --define:process.env.npm_package_name=\\\"$npm_package_name\\\" --define:process.env.npm_package_version=\\\"$npm_package_version\\\"",
     "bundle:pnpx": "esbuild lib/pnpx.js --bundle --platform=node --outfile=dist/pnpx.cjs",
     "bundle": "pnpm bundle:pnpm && pnpm bundle:pnpx",
-    "start": "pnpm tsc -- --watch",
+    "start": "pnpm tsc --watch",
```

Comment:
why is this needed?

Reply:
because it doesn't work when add `--`, so i removed it ...